pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -61,10 +62,25 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 			"systemDefaultRegistry": settings.SystemDefaultRegistry.Get(),
 		},
 	}
-	return setting, h.manager.Ensure(fleetChart.ReleaseNamespace, fleetChart.ChartName,
-		map[string]interface{}{
-			"apiServerURL": settings.ServerURL.Get(),
-			"apiServerCA":  settings.CACerts.Get(),
-			"global":       systemGlobalRegistry,
-		})
+
+	fleetChartValues := map[string]interface{}{
+		"apiServerURL": settings.ServerURL.Get(),
+		"apiServerCA":  settings.CACerts.Get(),
+		"global":       systemGlobalRegistry,
+	}
+
+	gitjobChartValues := map[string]interface{}{}
```

Comment:
```suggestion
	gitjobChartValues :=  make(map[string]interface{})
```
Use `make` when creating a new empty map